cobalt/doc/versioning.md

```diff
@@ -2,15 +2,19 @@
 
 Cobalt versions, as they appear in the user agent, have the following structure:
 
-**[Feature Year]**.**[Purpose]**.**[Update Number]**.**[Build ID]**
+**[Feature Year]**.**[Purpose]**.**[Update Number]**
```

Comment:
Let's use updated language:
Cobalt versions, as they appear in the user agent, use semantic versioning and have the following structure:
**[Major Version]**.**[Purpose]**.**[Minor Version]**

Reply:
Should we update cobalt/version.h as well to sync the version scheme?

Reply:
It is now [Major Version].[Purpose].[Minor Version].[Build ID]